packages/synapse-react-client/src/components/HeaderCard.tsx

```diff
@@ -118,11 +121,20 @@ const HeaderCard = forwardRef(function HeaderCard(
         <div className="row">
           <div className="col-md-offset-1 col-md-10">
             <div className="SRC-portalCardMain">
-              {!hideIcon && icon}
-              <div
-                style={{
+              <Box
+                sx={{
+                  ...(doiUri && { alignSelf: 'center', marginRight: '5px' }),
+                }}
+              >
+                {!hideIcon && icon}
+              </Box>
+              <Box
+                sx={{
                   width: '100%',
-                  ...(hideIcon && { display: 'flex' }),
+                  ...((doiUri || hideIcon) && {
+                    display: 'flex',
+                    justifyContent: 'space-between',
+                  }),
```

Comment:
If you can move the `CitationPopover` component to just slot into the `cardTopButtons`, I think you will probably be able to remove some of these conditional styles.